streams/src/main/java/org/apache/kafka/streams/processor/internals/TaskManager.java

```diff
@@ -354,11 +356,27 @@ void handleLostAll() {
         }
     }
 
+    /**
+     * @return Map from task id to that task's overall lag across all state stores
+     */
+    public Map<TaskId, Integer> getTaskOffsetSums() {
+        final Map<TaskId, Integer> taskOffsetSums = new HashMap<>();
+
+        for (final TaskId id : tasksOnLocalStorage()) {
+            if (isActive(id)) {
+                taskOffsetSums.put(id, ACTIVE_TASK_SENTINEL_OFFSET);
```

Comment:
```suggestion
                taskOffsetSums.put(id, -1);
```

No need for a constant, and it's more readable inline. Although, it could use a comment explaining the choice of sentinel values.

Reply:
Do you stick by this after seeing where it should be used below? Either way I'll add a comment

Reply:
No, I'm good now.